apps/frontend/src/context/DataContext.tsx

```diff
@@ -67,6 +74,34 @@ const zap: Waypoint[] = [
   { lat: 59.8856822, lon: 10.65318595, distance: 2000 },
 ];
 
+const SECONDS_30 = 30000;
+
+const useSaveDataToLocalStorage = () => {
```

Comment:
Would be nice with own file maybe?
but need the Data type. That could ofc be moved i guess